tests/yamls/agents/code_agent.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: maestro/v1alpha1
+kind: Agent
+metadata:
+  name: code
+  labels:
+    app: slack-example
+spec:
+  model: dummy
+  framework: code
```

Comment:
love this `code` as framework. Maybe we should have `language: python` since we could in theory have javascript? but OK to leave that as comment for future.